dashboard/app/auth.go

```diff
@@ -29,31 +29,42 @@ package main
 import (
 	"encoding/json"
 	"io/ioutil"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
 
 	"github.com/google/syzkaller/dashboard/dashapi"
-	"golang.org/x/net/context"
-	"google.golang.org/appengine/log"
 )
 
 const (
-	tokenInfoEndpoint = "https://oauth2.googleapis.com/tokeninfo"
+	// The official google oauth2 endpoint.
+	googleTokenInfoEndpoint = "https://oauth2.googleapis.com/tokeninfo"
 	// Used in the config map as a prefix to distinguish auth identifiers from secret passwords
 	// (which contain arbitrary strings, that can't have this prefix).
 	oauthMagic = "OauthSubject:"
 )
 
+// Represent a verification backend.
+type authEndpoint struct {
+	// URL supporting tokeninfo auth2 protocol.
+	url string
+	// TODO(blackgnezdo): cache tokens with a bit of care for concurrency.
+}
+
+func mkAuthEndpoint(u string) authEndpoint {
```

Comment:
Please call this `makeAuthEndpoint`. The standard prefixes for constructors are `make` and `new`, never seen `mk`.